src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -57,9 +57,6 @@ internal static Collection<FunctionBinding> GetBindings(ScriptHostConfiguration
                             }
                             bindings.Add(new HttpBinding(config, bindingMetadata, FileAccess.Write));
                             break;
-                        case "httptrigger":
```

Comment:
As you can see, I'm not removing the HttpBinding that exists for the HttpResponseMessage binding in this changeset. That will require further thought, and can be done in subsequent iterations. It's pretty tricky.